runtime/src/main/kotlin/kotlin/collections/ArrayUtil.kt

```diff
@@ -237,6 +237,22 @@ external private fun copyImpl(array: IntArray, fromIndex: Int,
 external private fun copyImpl(array: LongArray, fromIndex: Int,
                               destination: LongArray, toIndex: Int, count: Int)
 
+@SymbolName("Kotlin_ByteArray_copyImpl")
```

Comment:
Maybe add comment that operation is bitwise identical to signed types?

Reply:
Done